libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -924,6 +1194,35 @@ impl NodeAPI for Greenlight {
         }
     }
 
+    async fn max_sendable_amount(
+        &self,
+        payee_node_id: Option<Vec<u8>>,
+        max_hops: u32,
+        last_hop_hints: Option<&RouteHintHop>,
```

Comment:
nit: singular for now.
```suggestion
        last_hop_hint: Option<&RouteHintHop>,
```

Reply:
fixed